easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.3-foss-2015a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'PostgreSQL'
+version = '9.5.3'
+
+homepage = 'http://www.postgresql.org'
+description = """Client-side programs and libraries for accessing PostgreSQL databases."""
+
+source_urls = ['http://ftp.postgresql.org/pub/source/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+osdependencies = [('openssl-devel', 'libssl-dev')]
+
+dependencies = [
+    ('libreadline', '6.3'),
+    ('zlib', '1.2.8'),
+]
+
+builddependencies = [
+    ('Bison', '3.0.4'),
+    ('flex', '2.6.0'),
+    ('Perl', '5.22.0'),
+]
+
+configopts = '--with-openssl'
+
+# Client side only
+preinstallopts = 'make -C src/bin install && make -C src/include install && make -C src/interfaces install #'
```

Comment:
@valtandor We've installed the whole shabang up until now, so I'd stick to doing that; or introduce a `-client` `versionsuffix` in here.

If you go with the latter, include the URL as a comment above this `preinstallopts` line.